src/geo/map/cartodb-layer.js

```diff
@@ -10,13 +12,20 @@ var CartoDBLayer = LayerModelBase.extend({
   },
 
   initialize: function (attrs, options) {
+    attrs = attrs || {};
     LayerModelBase.prototype.initialize.apply(this, arguments);
     options = options || {};
 
     this._map = options.map;
     if (attrs && attrs.cartocss) {
       this.set('initialStyle', attrs.cartocss);
     }
+
+    this.infowindow = new InfowindowTemplate(attrs.infowindow);
```

Comment:
This is the "property" that layers expose as the infowindow API, so I think `.infowindow` is fine. Is not view, nor a model... it's a template! :)